### PR TITLE
Docs: Move to let/const, and clean up.

### PR DESCRIPTION
**Pull request by linbingquan**
Follow-up of #20484.

I made two commits, I hope it can helpful to review.

### CODE REVIEW

File: docs/api/zh/core/EventDispatcher.html [hunk 1]
```diff
@@ -20,19 +20,15 @@ <h2>代码示例</h2>
 		<code>
 		// 为自定义对象添加事件
 
-		const Car = function () {
+		const Car extends EventDispatcher () {
```
Comment (Mugen87) [inline docs/api/zh/core/EventDispatcher.html:23]: It should be just:
```js
const Car extends EventDispatcher {
```

Comment (linbingquan) [reply docs/api/zh/core/EventDispatcher.html]: > It should be:
> 
> ```js
> class Car extends EventDispatcher {
> ```

TBH, I coded it and after second just copy your code, made this mistake.
Thanks your reviewed, anyway, I changed it.

File: docs/api/zh/core/EventDispatcher.html [hunk 2]
```diff
@@ -20,7 +20,7 @@ <h2>代码示例</h2>
 		<code>
 		// 为自定义对象添加事件
 
-		const Car extends EventDispatcher () {
+		class Car extends EventDispatcher () {
```
Comment (Mugen87) [inline docs/api/zh/core/EventDispatcher.html:23]: You also have to remove `()`.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**